src/makeRobot.js

```diff
@@ -38,7 +38,50 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version}, `
+      + `wheels: ${this.wheels}`;
+    },
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(steps = 1) {
+      this.coords.y += Math.max(0, steps);
+
```

Comment:
No need to add 0. Fix all places
```suggestion
    goForward(steps = 1) {
      if (steps  > 0) {
        this.coords.y += steps  
      }

```